evergreen.yml

```diff
@@ -253,6 +273,8 @@ tasks:
       - func: "fetch packaged connector"
       - func: "sign archive"
       - func: "upload signed connector"
+      - func: "get signed connector"
+      - func: "verify signed connector"
```

Comment:
I created these two functions in order to verify that the signed `mongo_jdbc.taco` was put in s3 since it was unknown if Garasign created a signed file like the one created by Notary (i.e., `mongodb_jdbc-signed.taco`). I left this in to show how I verified it, but I can get rid of this before merging.

Reply:
You can add the `verify signed connector` after `sign archive` and you won't need to fetch it from s3 because it will then check it before uploading.  An alternative is you can add another step to `sign archive` and call it `sign and verify archive` and do the work there, as in add another  ` - command: shell.exec` step to sign archive.